cinema/serializers.py

```diff
@@ -85,9 +92,9 @@ class Meta:
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
-        Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], attrs["movie_session"]
-        )
+        Ticket.validate_ticket(attrs["row"],
+                               attrs["seat"],
+                               attrs["movie_session"])
```

Comment:
The `validate_ticket` method requires four arguments, but only three are provided here. Make sure to pass the `error_to_raise` argument as well to avoid runtime errors.